string pcd_path
---
geometry_msgs/PoseArray pose_array